src/App.js

```diff
@@ -8,15 +7,25 @@ class App extends Component {
   constructor(props) {
     super(props)
     this.state = {
+      currentUserKey: "92e8d1428fcadc1c114e4d05b203df9b",
       currentIngredient: null,
       recipeData: [],
-      calorieData: null,
+      mealHistory: {},
       isLoading: false,
       error: null,
       testing: "apples"
     }
   }
 
+  async componentDidMount(query) {
```

Comment:
I'd like to go through what's here with you at some point and describe what I think is happening and you can tell me if I'm totally wrong 😄